core/src/main/scala/org/ensime/core/SemanticHighlighting.scala

```diff
@@ -50,36 +50,41 @@ class SemanticHighlighting(val global: RichPresentationCompiler) extends Compile
           add(TypeParamSymbol)
         } else if (sym.hasFlag(PARAM)) {
           add(ParamSymbol)
-        } else if (sym.hasFlag(ACCESSOR)) {
-          val under = sym.accessed
-          if (under.isVariable) {
-            add(VarFieldSymbol)
-          } else if (under.isValue) {
-            add(ValFieldSymbol)
-          } else {
-            false
+        } else {
+          if (sym.isDeprecated) {
```

Comment:
can you do a standard `else if` here please? It'll avoid the indentation of everything else I think.